src/Export/Scripts/uniqueText.lua

```diff
@@ -0,0 +1,77 @@
+if not table.containsId then
+	dofile("Scripts/mods.lua")
+end
+
+local itemTypes = {
+	"axe",
+	"bow",
+	"claw",
+	"dagger",
+	"mace",
+	"staff",
+	"sword",
+	"wand",
+	"helmet",
+	"body",
+	"gloves",
+	"boots",
+	"shield",
+	"quiver",
+	"amulet",
+	"ring",
+	"belt",
+	"jewel",
+	"flask",
+}
+local catalystTags = {
+	["attack"] = true,
+	["speed"] = true,
+	["life"] = true,
+	["mana"] = true,
+	["resource"] = true,
+	["caster"] = true,
+	["attribute"] = true,
+	["physical"] = true,
+	["chaos"] = true,
+	["resistance"] = true,
+	["defences"] = true,
+	["elemental_damage"] = true,
+	["critical"] = true,
+}
+for _, name in pairs(itemTypes) do
+	local uniqueMods = LoadModule("../Data/Uniques/Special/Uniques.lua")
+	local out = io.open("../Data/Uniques/"..name..".lua", "w")
+	for line in io.lines("../Data/Uniques/Special/"..name..".lua") do
+		local specName, specVal = line:match("^([%a ]+): (.+)$")
+		if not specName and line ~= "]],[[" then
```

Comment:
It would be cleaner to read in the whole file as a Lua table, so you don't have to split by text anymore. This would also allow for simpler surrounding code.

Reply:
I actually don't know if this saves anything, after looking at it.  I have to parse each line of each item anyway, and the items would be all out of order if I parsed it as a table, meaning the diffs would be impossible to double-check for errors.